admin/controllers/service-manager.js

```diff
@@ -458,6 +461,96 @@ const controller = {
 
     req.flash('info', `'${school.name}' added as an MOD school`)
     return res.redirect(`/service-manager/mod-settings?hl=${school.urlSlug}`)
+  },
+
+  /**
+   * Manage service message
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+  getServiceMessage: async (req, res, next) => {
+    res.locals.pageTitle = 'Manage service message'
+    req.breadcrumbs(res.locals.pageTitle)
+    let serviceMessage
+    try {
+      serviceMessage = await serviceMessageService.getMessage()
+    } catch (error) {
+      return next(error)
+    }
+    res.render('service-manager/service-message/service-message-overview', {
+      breadcrumbs: req.breadcrumbs(),
+      serviceMessage
+    })
+  },
+
+  /**
+   * Submit service message
```

Comment:
No, it does not submit.

Reply:
@jon-shipley that was a bad paste. updated